components/team_selector_modal/team_selector_modal.tsx

```diff
@@ -246,6 +247,9 @@ export default class TeamSelectorModal extends React.PureComponent<Props, State>
         if (this.props.teams) {
             teams = this.props.teams.filter((team) => team.delete_at === 0);
             teams = teams.filter((team) => team.scheme_id !== this.currentSchemeId);
+            if (this.props.excludeGroupConstrained === true) {
+                teams = teams.filter((team) => team.group_constrained !== true);
```

Comment:
I believe this can be simplified to
```suggestion
                teams = this.props.excludeGroupConstained ? teams.filter((team) => !team.group_constrained) : teams;
```
and then you can remove the if statement.

Reply:
Changed!